Overview.md

```diff
@@ -0,0 +1,83 @@
+# Overview
+
+Learn a little bit more about Flare's firey APIs
+
+## LocalFirestore
+
+Flare's Local Firestore command allows for the reading, writing, and deleting of documents to be seamless and easy to use.
+There are a couple functions you can call to edit a document. SetDocument, GetDocumentOnce, and println() are all functions used to
+read, write, and display data values.
+
+```kotlin
+suspend fun main() {
+
+    @Serializable
+    data class Dog( //Class created to store id, name, age and string that is being written to the document

```

Comment:
Make sure all your comments have a space before them and start with a capital letter: `// My comment`